### PR TITLE
Fix #81: Clear duet committer when no args passed to git-solo

### LINKED ISSUE #81: git-solo with no args doesn't clear duet commiter
Hi, I use git-duet with GIT_DUET_CO_AUTHORED_BY so I care if `duet.env.git-committer-name` is set or not when doing solo programming.

The output from `git solo` is same as when I do `git solo ni` that I don't notice my typo, and the latter commit message is created with unwanted Co-Authored-by trailers.

What kind of fix is better for users?
For my usage I want to clear the committer when no args are given to git-solo.

Here's the example
```
$ mkdir tmp
$ cd tmp
$ git init
Initialized empty Git repository in /home/naoaki/src/tmp/.git/
$ cat ~/.git-authors 
authors:
    ni: Naoaki Iwakiri; naoaki
    hm: HHH MMM; him
    hr: HHH RRR; her
email:
  prefix:
  domain: example.com

$ git duet ni hm
GIT_AUTHOR_NAME='Naoaki Iwakiri'
GIT_AUTHOR_EMAIL='naoaki@example.com'
GIT_COMMITTER_NAME='HHH MMM'
GIT_COMMITTER_EMAIL='him@example.com'
$ git config --get-regexp 'duet.env' | cat
duet.env.git-author-initials ni
duet.env.git-author-name Naoaki Iwakiri
duet.env.git-author-email naoaki@example.com
duet.env.mtime 1568356020
duet.env.git-committer-initials hm
duet.env.git-committer-name HHH MMM
duet.env.git-committer-email him@example.com
$ git solo
GIT_AUTHOR_NAME='Naoaki Iwakiri'
GIT_AUTHOR_EMAIL='naoaki@example.com'
$ git config --get-regexp 'duet.env' | cat
duet.env.git-author-initials ni
duet.env.git-author-name Naoaki Iwakiri
duet.env.git-author-email naoaki@example.com
duet.env.mtime 1568356020
duet.env.git-committer-initials hm
duet.env.git-committer-name HHH MMM
duet.env.git-committer-email him@example.com
$ git solo ni
GIT_AUTHOR_NAME='Naoaki Iwakiri'
GIT_AUTHOR_EMAIL='naoaki@example.com'
$ git config --get-regexp 'duet.env' | cat
duet.env.git-author-initials ni
duet.env.git-author-name Naoaki Iwakiri
duet.env.git-author-email naoaki@example.com
duet.env.mtime 1568356114
duet.env.git-committer-initials 
duet.env.git-committer-name 
duet.env.git-committer-email 
```

### PR DESCRIPTION
**Pull request by naokiri**
Suggested solution on Issue #81.

I think we need to write test before merging when this direction is OK.

### CODE REVIEW

**Comment by Kehrlann** [comment]
On a side note, watch out, it seems running the tests changed your git config. Both your commits are by "Frances Bar".

Btw you can avoid these kind of issues by running `./scripts/test-docker`.

**Comment by naokiri** [comment]
Thanks for your note, I didn't realize that the test changed my git config. I'll use dockerised test next time.

File: configuration.go [hunk 1]
```diff
@@ -52,6 +55,10 @@ func NewConfiguration() (config *Configuration, err error) {
 		return nil, err
 	}
 
+	if config.PrecedingUpdateRequireInitials, err = strconv.ParseBool(getenvDefault("GIT_DUET_PRECEDING_UPDATE_REQUIRE_INITIALS", "0")); err != nil {
```
Comment (jszwedko) [inline configuration.go]: I'm wondering if we could just call the environment variable `GIT_DUET_DEFAULT_UPDATE`. I think that is a little more clear.

**Review by jszwedko — COMMENTED** [review]
@naokiri this looks great, thank you for making the updates and apologies for the repeated delay in my review. 

Just one small comment about the env var name, but otherwise I think this is good to go.

**Comment by Kehrlann** [comment]
Nice 👏 ! I would document this behavior in the README.

File: configuration.go (hunk 1 above)
Comment (naokiri) [reply configuration.go]: Sounds better. I'll find my time to update the PR later.

Comment (naokiri) [reply configuration.go]: Changed the environment variable name :+1:

**Comment by jszwedko** [comment]
Thanks for the updates @naokiri ! This looks good to me.

**Pull request merged**